examples/helm-on-kubernetes-example/readme.md

```diff
@@ -0,0 +1,53 @@
+# Helm Example 
```

Comment:
This `Helm example` content should be aligned with `NOTES.txt` content

Reply:
Done